eng/Versioning.targets

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <_BuildNumber>$(OfficialBuildId)</_BuildNumber>
     <_BuildNumber Condition="'$(OfficialBuildId)' == ''">$([System.DateTime]::Now.ToString(yyyyMMdd)).1</_BuildNumber>
+    <OriginalVersion>$(Version)</OriginalVersion>
```

Comment:
Why is this needed? It seems like you are only using it for release builds and in which case the Version should be unchanged.

Reply:
Let me check.

Reply:
`SkipDevBuildNumber` is not base on configuration, it's set by some CI builds only.

Reply:
Correct but if we are setting a dev version we don't have to error, in fact for our checkcode script should never be passing that.

Reply:
I wan't a check to verify that GenerateAPIListing isn't set to false in projects with release version set. How do I do it without storing the original version?

Reply:
You can use just the `Version` property. If it is not a dev build it will be exactly what you want and if it is a dev build then we aren't releasing it so it doesn't matter.

Reply:
> You can use just the Version property. If it is not a dev build it will be exactly what you want and if it is a dev build then we aren't releasing it so it doesn't matter.

I want it to fail in Analyze target in PR and Export-API.ps1 call . We are not releasing at that point.

Reply:
We are also not passing the dev build option during this CodeChecks.ps1 you are in control of that so it shouldn't ever be in that state. In fact if you want Export-APIs.ps1 to never be in that case you can explicitly pass `/p:SkipDevBuildNumber=false`